web/app.js

```diff
@@ -532,7 +532,7 @@ const PDFViewerApplication = {
       pdfLinkService.setHistory(this.pdfHistory);
     }
 
-    if (appConfig.findBar && !this.supportsIntegratedFind) {
+    if (!this.supportsIntegratedFind && appConfig.findBar) {
```

Comment:
This is technically unrelated to the other changes in the patch, but for the "full" Firefox PDF Viewer we skip one check this way :-)